jwst/regtest/test_nirspec_fs_spec3.py

```diff
@@ -52,10 +52,12 @@ def test_nirspec_fs_spec3(run_pipeline, rtdata_module, fitsdiff_default_kwargs,
         dmt = datamodels.open(rtdata.truth)
         dmr = datamodels.open(rtdata.output)
         if isinstance(dmt, datamodels.MultiSlitModel):
-            for slit_idx, slit in enumerate(dmt.slits):
+            names = [s.name for s in dmt.slits]
+            for name in names:
                 w = slit.meta.wcs
```

Comment:
I'm not sure this will work, since `slit` is not defined yet.
I think what's needed is
```
for name in names:
    slit_wcs, slit_wave = [(s.wcs, s.wavelength) for s in dmt.slits if s.name==name][0]
    _, _, wave = slit_wcs(x, y)
    assert_allclose(wave, slit_wave, atol=tolerance)
```

`assert_allclose` is imported from numpy.testing.